docs/docs/how-to/cloud/create-site-from-repository.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Create a Site from a Repository"
+description: "How to create a Gatsby Cloud site from a repository"
+---
+
+## Introduction
+
+Gatsby Cloud allows you to create a site by importing your Gatsby project from a Git repository. This guide provides the steps for importing a repository from your Git provider and building that site on Gatsby Cloud.
+
+## Prerequisites
+
+You can use the [gatsby-starter-blog](https://github.com/gatsbyjs/gatsby-starter-blog) as a starting point. Select the "Use this template" button to create a new repository using the starter as a template.
+
+![Button to create a new repository from a starter](../../images/use-this-template.png)
+
+## Directions
+
+### 1. Select a git provider
+
+From your Gatsby Cloud dashboard, click the **Add a Site** button.
+
+![Button to add a site from the Gatsby Cloud dashboard](../../images//add-site.png)
+
+In the "Import from a Git repository" section, choose your Git provider and install the Gatsby Cloud app. For the sake of this article you'll use GitHub, however GitLab and Bitbucket are also supported.
```

Comment:
The proper noun for GitHub is `GitHub` and in some other places of the added docs we have `Github`. So make sure that those are all consistent and correct 👍  Same applies to `GitLab` and `BitBucket`